### ISSUE TITLE
Slider controls are not clearly visible in high contrast mode

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4652]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.3 Contrast (Minimum)

**Test Environment:**
- Test URL: https://clarity.design/documentation/timeline
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Enable high contrast mode on windows (left shift + left alt + Print Screen key combo).
2. Open the test URL and observe the slider controls.

**Expected result:**
Slider controls should be clearly visible in high contrast mode.

**Actual result:**
Slider controls are partially visible which makes it hard to understand the control type.

**Comment by gkhadK** [comment]
In the normal viewing mode the slider bar has low contrast against the background, which also impacts low vision user. Improving the overall contrast for the slider bar, might fix both the issues. Using dark color border for the slider bar can resolve this issue all together. Reference: https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-1.html
